_partials/_devops-cli-get-started.md

```diff
@@ -0,0 +1,217 @@
+|         | whoami                           | Show information about the current user                                                        |
+| version |                                  | Show information about the currently installed version of $CLI_LONG                            |
+| config  |                                  | Manage your $CLI_LONG configuration                                                            |
+|         | show                             | Show the current configuration                                                                 |
+|         | set `<key>` `<value>`            | Set a specific value in your configuration. For example, `tiger config set debug true`         |
+|         | unset `<key>`                    | Clear the value of a configuration parameter. For example, `tiger config unset debug`          |
+|         | reset                            | Reset the configuration to the defaults. This also logs you out from the current $PROJECT_LONG | 
+| service |                                  | Manage the $SERVICE_LONGs in this $PROJECT_SHORT                                               |
+|         | describe `<service-id>`          | Show detailed information about a specific $SERVICE_SHORT in this $PROJECT_SHORT                      |
+|         | list                             | List all the $SERVICE_SHORTs in this $PROJECT_SHORT                                                   |
+|         | create                           | Create a new $SERVICE_SHORT in this $PROJECT_SHORT                                                    |
+|         | delete `<service-id>`            | Delete a $SERVICE_SHORT from this $PROJECT_SHORT                                                      |
+|         | update-password `<service-id>`   | Update the password for a $SERVICE_SHORT                                                       |
+| db      |                                  | Database operations and management                                                             |
+|         | connection-string `<service-id>` | Retrieve the connection string for a $SERVICE_SHORT                                            |
+|         | connect `<service-id>`           | Connect to a $SERVICE_SHORT                                                                    |
+|         | test-connection `<service-id>`   | Test the connectivity to a $SERVICE_SHORT                                                      | 
+| mcp     |                                  | Manage the $MCP_LONG                                                                           |
+|         | start                            | Start the $MCP_LONG                                                                            |
+|         | start `stdio` \| `http`          | Start the $MCP_LONG with stdio or HTTP transport.                                              |
```

Comment:
Note that we also have an [open PR](https://github.com/timescale/tiger-cli/pull/26) that adds a `tiger mcp install` command.